statsmodels/tsa/base/tsa_model.py

```diff
@@ -31,14 +31,14 @@
 _generic_params = base._model_params_doc
 _missing_param_doc = base._missing_param_doc
 
-class TimeSeriesModel(base.LikelihoodModel):
+class SeriesModel(base.LikelihoodModel):
```

Comment:
Yeah, sorry about that. The unit tests still passed after this, and I didn't read the global replace output carefully enough.